client/src/pages/Experience/Experience.js

```diff
@@ -2,26 +2,39 @@ import React, { useState, useEffect } from "react";
 import SignUpContainer from "../../components/SignUpContainer";
 import NewExperienceForm from "./NewExperienceForm";
 import AddExperienceButton from "./AddExperienceButton";
+import { availableLanguages } from "../../utils";
```

Comment:
can use absolute paths for the src root in these cases (I rarely like seeing ../, makes it harder in the future)